sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_autoinstrumentation/configurator.py

```diff
@@ -26,6 +33,17 @@ def _configure(self, **kwargs):
         if not _is_attach_enabled():
             warn(_PREVIEW_ENTRY_POINT_WARNING)
         try:
+            if environ.get(OTEL_TRACES_EXPORTER, "").lower().strip() != "none":
+                kwargs.setdefault("trace_exporter_names", ["azure-monitor-opentelemetry-exporter"])
+                try:
+                    sample_rate = float(environ.get("OTEL_TRACES_SAMPLER_ARG"))
+                except:  # pylint: disable=bare-except
+                    sample_rate = 1.0
+                kwargs.setdefault("sampler", ApplicationInsightsSampler(sample_rate))
+            if environ.get(OTEL_METRICS_EXPORTER, "").lower().strip() != "none":
+                kwargs.setdefault("metric_exporter_names", ["azure-monitor-opentelemetry-exporter"])
+            if environ.get(OTEL_LOGS_EXPORTER, "").lower().strip() != "none":
+                kwargs.setdefault("log_exporter_names", ["azure-monitor-opentelemetry-exporter"])
             super()._configure(**kwargs)
```

Comment:
Might be good to leave a comment stating `_configure` in upstream OpenTelemetry will treat exporters passed into `kwargs` as additive to the ones specified in env var so we know what to expect (as upstream can change implementation details in the future).

Reply:
Done.